drivers/eeprom/Kconfig

```diff
@@ -43,13 +43,17 @@ config EEPROM_AT2X
 
 config EEPROM_AT24
 	bool "Atmel AT24 (and compatible) I2C EEPROM support"
+	default y
+	depends on DT_HAS_ATMEL_AT24_ENABLED
 	depends on I2C
 	select EEPROM_AT2X
 	help
 	  Enable support for Atmel AT24 (and compatible) I2C EEPROMs.
 
 config EEPROM_AT25
 	bool "Atmel AT25 (and compatible) SPI EEPROM support"
+	default y
+	depends on DT_HAS_ATMEL_AT24_ENABLED
```

Comment:
should this be `DT_HAS_ATMEL_AT25_ENABLED`?

Reply:
Good catch, thanks. Fixed.